src/Cli/dotnet/commands/dotnet-workload/list/WorkloadListCommand.cs

```diff
@@ -115,17 +116,20 @@ public override int Execute()
             }
             else
             {
+                InstalledWorkloadsCollection www = new(installedList, $"SDK {_currentSdkFeatureBand}");
```

Comment:
Nit: Why `www` for a variable name?

Reply:
Thanks, I thought I renamed it. I used a temp name when I couldn't come up with something good, but thought I renamed everything. I'll fix this.